File: viajerosrepojava-master/viajerosrepojava-master/viajeros/viajeros/src/main/java/ps/com/viajeros/dtos/viaje/SearchResultMatchDto.java
package ps.com.viajeros.dtos.viaje;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class SearchResultMatchDto {
    private Long tripId;
    private String origin;
    private String destination;
    private int availableSeats;
    private LocalDateTime date;
    private LocalDateTime departureTime;
    private LocalDateTime arrivalTime;
    private String estimatedDuration;
    private boolean petsAllowed;
    private boolean smokersAllowed;
    private String vehicleName;
    private double driverRating;
    private String driverName;
    private Long driverId;
    private String status;

    // Getters y setters
}

File: viajerosrepoangular-master/viajerosrepoangular-master/viajeros/src/app/models/Viajes/SearchResultMatchDto.ts
export interface SearchResultMatchDto {
    tripId: number;
    origin: string;
    destination: string;
    availableSeats: number;
    date: Date; // Considera usar Date si necesitas manipular fechas
    departureTime: Date; // Puedes usar Date o string según tus necesidades
    arrivalTime: Date; // Puedes usar Date o string según tus necesidades
    estimatedDuration: string;
    petsAllowed: boolean;
    smokersAllowed: boolean;
    vehicleName: string;
    driverRating: number; // Suponiendo que sea un valor numérico
    driverId: number;
    driverName: string;
    status:string;
  }
  